cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/ComputerVision.scala

```diff
@@ -454,8 +454,6 @@ class GenerateThumbnails(override val uid: String)
     new CustomOutputParser().setUDF({ r: HTTPResponseData => r.entity.map(_.content).orNull })
```

Comment:
Since this class overrides the output parser without calling super, the behavior or HasInternalJsonOutputParser is ignored

Reply:
thanks for finding this!